airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -603,7 +614,7 @@ def _try_get_error(value: Any) -> Any:
             return None
 
         try:
-            body = response.json()
+            body = next(decoder.decode(response))
```

Comment:
is there a chance decoder.decode will return an empty iterable? If yes, let's catch the exception and raise a user-friendly error message